.rubocop_todo.yml

```diff
@@ -1,11 +1,21 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2020-10-30 22:23:14 UTC using RuboCop version 0.93.1.
+# on 2020-11-02 23:35:33 UTC using RuboCop version 0.93.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 2
+# Configuration parameters: IgnoredMethods.
+Metrics/AbcSize:
+  Max: 27
```

Comment:
I would really like to avoid setting these thresholds automatically.  Instead by wrapping each method with an exception, we avoid the case where all our methods approach the worst case.

Reply:
will change to an exclude if that's ok